### DISCUSSION TITLE
registering  3-D FLAIR  images using pre-trained models

### DISCUSSION THREAD

**Discussion by Ehsan-data** [voxelmorph/voxelmorph#410] [Q&A]
Hi there,

I wonder if using the pre-trained models would create any issue when registering  3-D FLAIR  images instead of T1 weighted images. I used the pre-trained model "brain_3d.h5" to register my MRI images, and the results were not good. 

Also, come to my mind if my image intensity needs to be within a particular range?

Cheers

**Comment by adalca** [comment]
I would consider trying the synthmorph pre-trained models, which @mu40 can point you to. More info here: http://synthmorph.voxelmorph.net/

**Comment by Ehsan-data** [comment]
Thank you. Would you please advise if the image intensity is essential? If yes, would you recommend a preferred way of bringing image intensities within the acceptable range?

**Comment by adalca** [comment]
sure, just normalize to [0,1] when using synthmorph, I think. @mu40 ?

**Reply by mu40** [reply]
That's correct. The models expect the intensities of each image to be min-max normalized to [0, 1], or they may misbehave. For best performance, you may also want to consider aligning your input images with the reference brain as explained at https://github.com/voxelmorph/voxelmorph#SynthMorph (if you choose the model trained on brains).

**Reply by Ehsan-data** [reply]
Cheers for that.
